README.md

```diff
@@ -1 +1,39 @@
-# Yahtzee
\ No newline at end of file
+
+<h1>Yahtzee</h1>
+<h2>Introduction</h2>
+*Yahtzee* is a chance and strategic dice game. In this game participants can score points by rolling five dice. The players of the game have to decide whether they want to combine the dice into a score, or whether they roll the dice again. The dice can be rolled up to three times in one turn. The scoring categories (see: [Upper Section Categories](https://en.wikipedia.org/wiki/Yahtzee#Upper_section) and [Lower Section Categories](https://en.wikipedia.org/wiki/Yahtzee#Lower_section)) have varying point values, some of which are fixed and others have to be calculated. The winner of the game is the one with most points.
+
+According to [MoSCoW](https://en.wikipedia.org/wiki/MoSCoW_method) method: 
+
+*Must have* – short on time| *Should have* – goal  | *Could have* – extra time  | *Won’t have* n/a  
```

Comment:
Great schedule! It's also possible to use an animation for throwing the dice (something you could spend more or less time on).

Reply:
Likewise, if you have extra time you could also add funny animations for special results, like [bowling alleys](https://www.youtube.com/watch?v=ZgYPPZS7Fb0) do.